api-reference/receipts/endpoints.markdown

```diff
@@ -0,0 +1,672 @@
+```
+
+_Example Response:_
+
+```shell
+HTTP/1.1 202 Accepted
+Link: <https://us.api.concursolutions.com/receipts/v4/status/b0a4ab2bce8a49a08cf177cb997bf2ee>; rel="processing-status"
+Location: https://us.api.concursolutions.com/receipts/v4/images/b0a4ab2bce8a49a08cf177cb997bf2ee
+Content-Length: 0
+Connection: keep-alive
+```
+
+[Back to Top](#endpoints)
+
+##### Endpoint: Get Image-Only Receipts By UserId
+
+###### _GET /v4/users/:userId/image-only-receipts_
+
+|Parameter|Requirement|Value|
+|---|---|---|
+|userId|required|The UUID of the user whose receipt images will be returned.|
+
+Returns the JSON metadata of receipt images for the user ID specified in the URL. Results should be paginated in the same manner as the eReceipt endpoint.
+
+_Example Requests:_
+
+cURL:
+
+```shell
+curl -H "Authorization: Bearer {YOUR ACCESS TOKEN}" https://us.api.concursolutions.com/receipts/v4/users/{USER ID}/image-only-receipts
+```
+
+HTTPie:
+
+```shell
+http https://us.api.concursolutions.com/receipts/v4/users/{USER ID}/image-only-receipts "Authorization: Bearer {YOUR ACCESS TOKEN}"
+```
+
+_Example Response:_
+
+```json
+HTTP/1.1 200 OK
+Content-Length: 800
+Connection: keep-alive
+
+{
+	"receiptsImages": [
+	    {
+	        "dateTimeReceived": "Wed May 24 2017 16:14:17 GMT+00:00",
+	        "id": "a90fc48e0f0a44f2bd4838fd773b07a5",
+	        "image": "https://us.api.concursolutions.com/receipts/v4/image-only-receipts/a90fc48e0f0a44f2bd4838fd773b07a5/image",
+	        "imageId": "2B7095DA565D3EB9AC82B6712FCC0F79",
+	        "userId": "abcd123456efg"
+	    },
+	    { ... },
+	],
+	"next": "https://us.api.concursolutions.com/receipts/v4/users/abcd123456efg/image-only-receipts/page/1507587575592_d4721b2f3b304a9a9325fabdad5f50ad"
+}
+```
+
+[Back to Top](#endpoints)
+
+##### Endpoint: Get an Image-Only Receipt by ID
+
+###### _GET /v4/image-only-receipts/:receiptId_
+
+|Parameter|Requirement|Value|
+|---|---|---|
+|receiptId|required|The UUID of the receipt image to be returned.|
+
+Returns the JSON metadata associated with the ID in the URL.
+
+_Example Requests:_
+
+cURL:
+
+```shell
+curl -v -X GET https://us.api.concursolutions.com/receipts/v4/image-only-receipts/{RECEIPT ID} \
+-H "Authorization: Bearer {YOUR ACCESS TOKEN}"
+```
+
+HTTPie:
+
+```shell
+http https://us.api.concursolutions.com/receipts/v4/image-only-receipts/{RECEIPT ID} "Authorization: Bearer {YOUR ACCESS TOKEN}"
+```
+
+_Example Response_
+
+```json
+HTTP/1.1 200 OK
+Content-Length: 272
+Connection: keep-alive
+
+{
+    "dateTimeReceived": "Wed May 24 2017 16:14:17 GMT+00:00",
+    "id": "a90fc48e0f0a44f2bd4838fd773b07a5",
+    "image": "https://us.api.concursolutions.com/receipts/v4/image-only-receipts/a90fc48e0f0a44f2bd4838fd773b07a5/image",
+    "imageId": "2B7095DA565D3EB9AC82B6712FCC0F79",
+    "userId": "abcd123456efg"
+}
+```
+
+[Back to Top](#endpoints)
+
+##### Endpoint: Get Receipt Image (Image-Only)
+
+###### _GET /v4/image-only-receipts/:receiptId/image-only-receipts_
```

Comment:
Should be `GET /v4/image-only-receipts/:receiptId/image`

Reply:
✅